docs/user/customization.md

```diff
@@ -0,0 +1,122 @@
+
+An example `install-config.yml` is shown below. This configuration has been modified to show the customization that is possible via the install config. By default, the generated `install-config.yml` will not have all of these fields populated, so they will need to be manually added if they are needed.
+
+```yaml
+apiVersion: v1beta1
+
+pullSecret: "auths": ...
+sshKey: ssh-ed25519 AAAA...
+
+baseDomain: example.com
+metadata:
+  name: test-cluster
+
+machines:
+- name: master
+  replicas: 3
+  platform:
+    zones:
+    - us-west-2a
+    - us-west-2b
+- name: worker
+  replicas: 5
+  platform:
+    type: c5.9xlarge
+    iamRoleName: elastictranscoder-access
+    rootVolume:
+      iops: 4000
+      size: 500
+      type: io1
+    zones:
+    - us-west-2c
+
+networking:
+  clusterNetworks:
+  - cidr: 10.128.0.0/14
+    hostSubnetLength: 9
+  machineCIDR: 10.0.0.0/16
+  serviceCIDR: 172.30.0.0/16
+  type: OpenshiftSDN
+
+platform:
+  aws:
+    region: us-west-2
+    userTags:
+      costCenter: 7536
+      adminContact: jdoe
+```
+
+#### AWS Platform Customization
+
+The following options are available when using AWS:
+
+- `machines.platform.iamRoleName` - the IAM role that will be assigned to the machines of this pool
+- `machines.platform.rootVolume.iops` - the reserved IOPS of the root volume
+- `machines.platform.rootVolume.size` - the size (in gigabytes) of the root volume
+- `machines.platform.rootVolume.type` - the storage type of the root volume
+- `machines.platform.type` - the EC2 instance type
+- `machines.platform.zones` - a list of the availability zones that the installer will use when creating machines of this pool
+- `platform.aws.region` - the AWS region that the installer will use when creating resources
+- `platform.aws.userTags` - a map of keys and values that the installer will add as tags to all resources it creates
+
+### Kubernetes Customization (unmanaged)
+
+In addition to customizing OpenShift and aspects of the underlying platform, the installer allows arbitrary modification to the Kubernetes objects that are injected into the cluster. Note that there is currently no validation on the modifications that are made, so it is possible that the changes will result in a non-functioning cluster. The Kubernetes manifests can be viewed and modified using the `manifests` and `manifest-templates` targets.
+
+The `manifests` target will render the manifest templates and output the result into the asset directory. Perhaps the most common use for this target is to include additional manifests in the initial installation. These manifests could be added after the installation as a "Day 2" operation, but there may be cases where they are necessary beforehand.
+
+The `manifest-templates` target will output the pre-rendered manifest templates into the asset directory. This allows modification to the templates before they have been rendered, which may be useful to users who wish to reuse the templates between cluster deployments.
```

Comment:
"pre-rendered" feels ambiguous.  Maybe "unrendered", to avoid folks interpreting this as "rendered before I showed them to you"?